docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -120,12 +120,12 @@ IMPORTANT: The `msmarco-passagetest2019-top1000` dataset was not utilized to tra
 It is only used in this tutorial as a sample dataset that is easily accessible for demonstration purposes.
 You can use a different data set to test the workflow and become familiar with it.
 
-Download the file and upload it to your cluster using the
-{kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[Data Visualizer]
-in the {ml-app} UI.
-Assign the name `id` to the first column and `content` to the second column.
-The index name is `test-data`.
-Once the upload is complete, you can see an index named `test-data` with 182469 documents.
+Download the file and upload it to your cluster using the {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[Data Visualizer] in the {ml-app} UI.
```

Comment:
```suggestion
Download the file and upload it to your cluster using the {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[File Uploader] in the UI.
```

Search team have been using **File Uploader** term, so probably makes sense to align with that usage. See also [ES3 docs](https://www.elastic.co/docs/current/serverless/elasticsearch/ingest-data-file-upload).

Also minor suggestion to say "UI" instead of "ML UI"